### ISSUE TITLE
How to run test

### ISSUE THREAD

**Issue by da03** [google-research/long-range-arena#3]
This might be a dumb question, but it seems that there's only a train.py which trains and prints validation stats. How to test the model to get numbers comparable to accuracy numbers in the table?

**Comment by vanzytay** [comment]
Some tasks (like imdb) do not have validation sets, so the results on "val" are tests. On other tasks, you the test sets are usually also produced as a tensorflow dataset object. So you may simply replace "val" with "test" in the main training script or copy the eval loop once more to eval on test.

**Comment by cifkao** [comment]
Should we expect evaluation scripts/notebooks to be added soon?

**Comment by cifkao** [comment]
See #8 for a fix.

**Comment by BalloutAI** [comment]
@cifkao, the test_only is giving very low accuracy on listops, while the validation accuracy during training is much higher. Any idea why?
